DeAceroDLabs/marketplace-frontend
# Feature/dmp 641 Market on google maps

## What is done

[[DMP-641] Market on google maps](https://deacero.atlassian.net/browse/DMP-641)

I think this pr also covers DMP-642

[[DMP-642] Market on google maps](https://deacero.atlassian.net/browse/DMP-642)

## How is done

- Added functionality so that  map location changes depending on postal code.
- Added Marker to maps.
![image](https://user-images.githubusercontent.com/81311646/197646312-ef6b9e1f-0ec3-4a34-aa23-5d4ad75b87dc.png)

- Zoom updates.

## How to test

1. Run the project via `npm start`
2. Go to registrarse.
3. Fill first form.
4. On the second form map should be displayed and you should be able to see and change the marker with no problems.
